app.json

```diff
@@ -13,7 +13,11 @@
       "description": "development environment",
       "value": "development"
     },
-    "NPM_CONFIG_PRODUCTION": {
+    "HEROKU": {
```

Comment:
Can actually drop this. Core uses this for the threadloader within the webpack config.